doc/matlab_visualization.md

```diff
@@ -0,0 +1,43 @@
+# How to use Matlab Visualization
```

Comment:
```suggestion
# How to use MATLAB Visualization
```
The correct capitalization of MATLAB, see the official documentation https://nl.mathworks.com/products/matlab.html .